test/fixtures/receipt_mailer/receipt_email

```diff
@@ -15,10 +15,10 @@ either your ProQuest form, title page, and abstract or the electronic version of
 your September 2017 thesis "MyString" to the MIT Libraries.
 </p>
 
-<p>If you submitted a May 2020 theses, please note that only the ProQuest form,
-title page, and abstract should have been submitted here. The actual thesis
-should go directly to your department. For degree periods prior to May 2020, we
-accept all electronic copies via this form.</p>
+<p>If you submitted a May or September 2020 thesis, please note that only the
```

Comment:
This is the test for the receipt email in the first file above, so any change there needs to be repeated here